core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/options/IFramePlayerOptions.kt

```diff
@@ -66,6 +68,24 @@ class IFramePlayerOptions private constructor(private val playerOptions: JSONObj
             return this
         }
 
+        /**
+         * Controls if the video is played automatically.
```

Comment:
Change to something like: "Controls if the video is played automatically after the player is initialized"